raiden/transfer/views.py

```diff
@@ -571,7 +571,7 @@ def list_channelstate_for_tokennetwork_lumino(
     chain_state: ChainState,
     payment_network_id: PaymentNetworkID,
     token_addresses_split,
-    node_address
+    _node_address
```

Comment:
We should remove this parameter, its not used and it makes no sense since the method returns the tokens. Thats not related with a node address in particular.